flink-test-utils-parent/flink-connector-test-utils/src/main/java/org/apache/flink/connector/testframe/testsuites/SinkTestSuiteBase.java

```diff
@@ -508,6 +509,7 @@ private void checkResultWithSemantic(
                                 .matchesRecordsFromSource(Arrays.asList(sort(testData)), semantic);
                         return true;
                     } catch (Throwable t) {
+                        LOG.error("Ooops", t);
```

Comment:
Please use a more descriptive error message.

Reply:
Whoops, this was not supposed to be merged. The way the test is designed the exception is swallowed by default, this is a remainder of my debugging.

Reply:
Please notice that without this log, the tests will simply accidentally hang if you were to run them without "sleep" in the reader.